models/sightingModel.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose';
+
+const Sighting = mongoose.model('Sighting', {
+  date: Date,
```

Comment:
I think there should be some required fields in this schema. Date and location for sure 👽